### ISSUE TITLE
Select local Git 4 Windows installation during installation

### ISSUE THREAD

**Issue by SteveALee** [ForkIssues/TrackerWin#485]
When you 1st install (on WIndows) you are asked for user credentials. This s surprising when you allready set these in your global git config.

I see in the settings that Fork seems to have it's own Git for Windows (G4W) installation and I can select my own (more recent) G4W installation.

Can you alow selection of existing G4W installation during installation? Then it will not be necessary to reenter user credentials (with the possibility of error).

**Comment by DanPristupov** [comment]
> Can you allow selection of existing G4W installation during installation?

Yes, in the future. We have introduced the ability to switch git instance in Fork 1.40 just a week ago. Now we need to ensure it is works without problems for everybody.

**Comment by Lonli-Lokli** [comment]
Any update? It would be great to fetch global settings if they already exists
